src/cli/get-manifest.js

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env node
+const merge = require('deepmerge');
+const { unionMerge, readManifestJSONFile, readImportedManifestFile, hasManifest } = require('./hook-utils/manifest');
+
+/** 
+ * Implements the get-manifest script hook required by the Slack CLI
+ * Returns a manifest JSON string to stdout.
+ * Looks for a manifest.ts, .js and .json files. If multiple are found 
+ * Then they any manifest.ts (or missing that, manifest.js) defined
+ * properties will merge into any manifest.json
+*/
+(function _(cwd) {
+  const file = 'manifest';
+  let manifest = {};
+
+  // look for a manifest files
+  const manifestJSON = readManifestJSONFile(cwd, `${file}.json`);
+  const manifestTS = readImportedManifestFile(cwd, `${file}.ts`);
+  const manifestJS = readImportedManifestFile(cwd, `${file}.js`);
+
+  if (!hasManifest(manifestTS, manifestJS, manifestJSON)) {
+    throw new Error('Unable to find a manifest file in this project');
+  }
+
+  // manage manifest merge
+  // check for .json
+  if (manifestJSON) {
+    manifest = merge(manifest, manifestJSON, { arrayMerge: unionMerge});
```

Comment:
The default array merge strategy for the `deepmerge` library is concatenation. `unionMerge` provided as a custom merge strategy resolves this issue.

Reply:
@shapirone - `botScopes` is another good example. If you had a manifest.json and and manifest.js with scopes defined in each, you'd want those scopes not to be duplicated or manifest API complains at you. This issue exists in the current deno builder implementation actually! This scenario results in a manifest with repeated scopes.

Reply:
Makes sense, something we could fix in the builder library.